### PR TITLE
Feature/style sectors page

### PR DESCRIPTION
**Pull request by tsubik**
Sectors' index and show page styling V1

- Index page style refresh
- Show page style refresh and adding companies level ladder
- Adding real links to home page and navbar
- Adding stars!
- MQ assessment levels different colors for different levels

image

### CODE REVIEW

File: app/assets/stylesheets/tpi/_colors.scss [hunk 1]
```diff
@@ -1,5 +1,7 @@
 $white: #fff;
+$blue-light: #86A9F9;
 $blue: #5587F7;
+$blue2: #2465F5;
```
Comment (rapzo) [inline app/assets/stylesheets/tpi/_colors.scss]: Don't use numbers for colors. The more verbose the better to understand where to use.

File: app/assets/stylesheets/tpi/_typography.scss [hunk 2]
```diff
@@ -14,18 +14,22 @@
 
 h1 {
   @extend .is-size-1;
+  font-weight: bold;
```
Comment (rapzo) [inline app/assets/stylesheets/tpi/_typography.scss]: I've added almost the same thing but don't use weight, use the bold font variation.

File: app/assets/stylesheets/tpi/_colors.scss (hunk 1 above)
Comment (tsubik) [reply app/assets/stylesheets/tpi/_colors.scss]: Yeah, I know, couldn't find a good name for a color between blue and blue-dark :D

Comment (tsubik) [reply app/assets/stylesheets/tpi/_colors.scss]: Normally I use this http://chir.ag/projects/name-that-color/

Comment (tsubik) [reply app/assets/stylesheets/tpi/_colors.scss]: maybe we should use it for all colors?

File: app/assets/stylesheets/tpi/_typography.scss (hunk 2 above)
Comment (tsubik) [reply app/assets/stylesheets/tpi/_typography.scss]: Ok, I checked your PR and see what you mean

File: app/assets/stylesheets/tpi/_colors.scss (hunk 1 above)
Comment (rapzo) [reply app/assets/stylesheets/tpi/_colors.scss]: Sure. Was just a comment, not a blocker :-)

**Comment by tsubik** [comment]
Ready!

**Comment by aabdaab** [comment]
wow, nice! 🙌 starting to look pretty 💅 I'm gonna check the code now

**Comment by aabdaab** [comment]
small thing but we've got this unnecessary bottom border between the columns:
image

**Comment by aabdaab** [comment]
haven't found any issues but this bottom-border one.

**Comment by tsubik** [comment]
Good catch with the line @aabdaab

**Pull request merged**